README.md

```diff
@@ -40,6 +40,7 @@ The following arguments are supported:
 
 * `port` - (Required) Server port
 * `vpc` - (Required) The VPC name
+* `cidr_blocks_extra` - (Optional) List of additional CIDR blocks granted access to the server security group. Defaults to an empty list.
```

Comment:
I recommend you drop the word `extra` in the name `cidr_blocks_extra`. In the future we may want to add for example `cidr_blocks_ipv6`. The fact that the list is optional I think makes it clear enough that it is well extra.

I also recommend that you specify that it is inbound access in the description.

Reply:
Kala and I see your point, but we think the word "extra" clarifies the intent.

Regarding IPv6, I have been hearing about it for 15+ years, and yet it has never reached a state where it affects our work. All of our modules doing networking will need to be reworked if and when IPv6 becomes relevant to us, so we agreed to put this off until that day comes – and we might all be retired before that happens! :-)